src/main.js

```diff
@@ -0,0 +1,52 @@
+class Person {
+  constructor(name, birthday, amount) {
+    this.name = name;
+    this.birthday = birthday;
+    this.initialAmount = amount;
+    this.history = [`Initial: ${this.initialAmount}`];
```

Comment:
will be better if you saved transaction in `object` instead of `string`

Reply:
Done